fpsim/locations/kenya/kenya.py

```diff
@@ -0,0 +1,817 @@
+'''
+Set the parameters for FPsim, specifically for Kenya.
+'''
+
+import numpy as np
+import pandas as pd
+import sciris as sc
+from scipy import interpolate as si
+from fpsim import defaults as fpd
+
+
+# %% Housekeeping
+
+def scalar_pars():
+    scalar_pars = {
+        # Basic parameters
+        'location': 'senegal',
```

Comment:
change to Kenya :)

Reply:
done! This feels related to the syntax issues with calling parameters, but is now changed in kenya.py